crates/bevy_ecs/src/lib.rs

```diff
@@ -60,7 +60,10 @@ pub use bevy_ptr as ptr;
 ///
 /// This includes the most common types in this crate, re-exported for your convenience.
 pub mod prelude {
-    #[expect(deprecated)]
+    #[expect(
+        deprecated,
+        reason = "`crate::schedule::apply_deferred` is considered deprecated; however, it may still be used by crates which consume `bevy_ecs`, so its removal here may cause confusion."
```

Comment:
I really couldn't think of a better reason, but I also don't know why it's still here.

Reply:
This will be removed in the Bevy 0.17 cycle.

Reply:
I will edit the reason to mention this.